lib/render/multiple-files.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
+  const setEditorMainFile = () => {
+    const multipleFileEditor = mumuki.multipleFileEditor;
+    if(!multipleFileEditor) return setTimeout(setEditorMainFile, 100);
+    multipleFileEditor.mainFile = 'index.html';
```

Comment:
Here you could use the `mumuki.version` property to conditionally enable this feature.